COPYRIGHT.txt

```diff
@@ -344,6 +344,11 @@ Comment: Recast
 Copyright: 2009, Mikko Mononen
 License: Zlib
 
+Files: ./thirdparty/rvo2/
+Comment: RVO2
```

Comment:
What version of RVO is this? Is there a git hash or a version number?

Reply:
thirdparty/README.md